src/features/wallet/send/__test__/SendView.test.tsx

```diff
@@ -0,0 +1,50 @@
+import 'react-native'
+import React from 'react'
+import { createStackNavigator } from '@react-navigation/stack'
+import Balance, { CurrencyType } from '@helium/currency'
+import SendView from '../SendView'
+import { render, cleanup, fireEvent } from '../../../../utils/testUtils'
+import { createAccount } from '../../../../__fixtures__/account'
+
+const { Screen, Navigator } = createStackNavigator()
+
+afterEach(cleanup)
+
+it('renders correct account balance', async () => {
+  const { findByText } = render(
+    <Navigator>
+      <Screen name="SendView" component={SendView} />
+    </Navigator>,
+    {
+      initialState: {
+        account: {
+          account: createAccount({
+            balance: new Balance(79942876300, CurrencyType.networkToken),
+          }),
+        },
+      },
+    },
+  )
+  const text = await findByText('799.429 HNT Available')
+  expect(text).toBeDefined()
+})
+
+it('comma formats amount separator', async () => {
+  const { findByTestId } = render(
+    <Navigator>
```

Comment:
Do we need to pass a `Navigator` every time? Maybe we can put that inside the render and make it so that you can just pass the component? Would it support passing props to the component too?

Reply:
In this case, it is a Stack Navigator. My thought was that there might be situations where we want different types of navigators in the future.